README.md

```diff
@@ -90,6 +90,14 @@ jobs:
       line_length: 120
     EOF
     ```
+  
+1. Autofix code formatting errors using those linters with a built-in `fix` option.
```

Comment:
```suggestion
1. Autofix code formatting errors using those linters with a built-in `fix` option. Note, this will create a `dirty` git directory, and you will need to manage committing and pushing any changes; consider setting `INPUT_AUTO_FIX=true` locally instead.
```